### PR TITLE
Fix example llm recipes and autoround default value.

### PR DESCRIPTION
**Pull request by changwangss**
## Type of Change
1. autoround suggets the default values use as follows.
iters 200
samples 512
seqlen 2048
asym
lr None
minmax_lr None

2. due to example script changed, the llm recipes command and itrex commit follow changes, @XuehaoSun  is validating now.
## Description

detail description 
JIRA ticket: xxx

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

File: examples/huggingface/pytorch/text-generation/quantization/run_generation_cpu_woq.py [hunk 1]
```diff
@@ -83,7 +83,7 @@
     choices=["fp32", "bf16", "int8"],
 )
 parser.add_argument("--group_size", type=int, default=32)
```
Comment (WeiweiZhang1) [inline examples/huggingface/pytorch/text-generation/quantization/run_generation_cpu_woq.py:85]: can we change default group_size to 128

**Comment by WeiweiZhang1** [comment]
autoround args of run_generation_cpu_woq will also be aligned, right?

**Comment by changwangss** [comment]
> autoround args of run_generation_gpu_woq will also be aligned, right?

improved.

File: examples/huggingface/pytorch/text-generation/quantization/run_tuning.sh [hunk 2]
```diff
@@ -238,7 +238,7 @@ function run_tuning {
         script="run_generation_sq.py"
     elif [ "${topology}" = "llama2_7b_gptq" ]; then
         model_name_or_path="meta-llama/Llama-2-7b-hf"
-        extra_cmd=$extra_cmd" --woq --bits 4 --compute_dtype fp32 --scheme asym "
+        extra_cmd=$extra_cmd" --woq --bits 4 --compute_dtype fp32 --scheme asym --calib_iters 100"
```
Comment (wenhuach21) [inline examples/huggingface/pytorch/text-generation/quantization/run_tuning.sh:241]: why 100, the official setting of gptq and awq  is 128, autoround is 512

Comment (changwangss) [reply examples/huggingface/pytorch/text-generation/quantization/run_tuning.sh:241]: it means calibration run inference iteration number.
`autoround` use "iters", but it has another meaning that how many benchmark inference iterations to compute the throughput  in itrex, so use "calib_iters", and "calib_iters" also is used by smoothquant before, it is the same meaning to express how many iterations number in calibration function.
`gptq` doesn't has this parameter.

**Pull request merged**